jobs/payment-jobs/services/email_service.py

```diff
@@ -25,30 +25,30 @@
 from pay_api.utils.enums import AuthHeaderType, ContentType
 
 
-def send_email(recipients: str, subject: str, body: str):
+def send_email(recipients: list[str], subject: str, body: str):
```

Comment:
fix to be in sync with how the email_service works in pay_api due to getting emails by keycloak role fix

Reply:
shouldn't we just put this into pay-api?

Reply:
sounds good, I can just extend the pay_api email service as a base class so it shares the same send_email and allow me to keep the template functions